.travis.yml

```diff
@@ -1,12 +1,41 @@
 language: go
-sudo: false
+
+sudo: required
+
+services:
+  - docker
+
 go:
   - 1.7.3
+
+before_install:
+  - export HostIP="192.168.12.50"
```

Comment:
etcd will be running on this internal ip address and service-center will be connect with etcd using this ip